### PR TITLE
feat: improve log viewer UI

### PR DESCRIPTION
**Pull request by Niraj-Kamdar**
- Closes: #93

### CODE REVIEW

File: packages/verifier/client/web-ui/src/types.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export type ClientInfo = {
```
Comment (nerfZael) [inline packages/verifier/client/web-ui/src/types.ts:1]: I know they're small, but it'd be nice if they were in their own files :)

File: packages/verifier/client/web-ui/src/App.tsx [hunk 2]
```diff
@@ -60,19 +53,31 @@ function App(): JSX.Element {
...
+        .catch((error) => {
+          console.log(error);
+        });
+    };
+
+    logFetcher();
+    console.log(3);
+
+    const logFetcherId = setInterval(logFetcher, 10000);
+    // setLogFetcherId(logFetcherId);
```
Comment (nerfZael) [inline packages/verifier/client/web-ui/src/App.tsx:76]: what's this?

File: packages/verifier/client/web-ui/src/App.tsx [hunk 3]
```diff
@@ -87,6 +92,33 @@ function App(): JSX.Element {
     },
   ];
 
+  function onSearch(e: React.ChangeEvent<HTMLInputElement>) {
+    setLogFilterOptions((options: LogFilterOptions) => {
+      const newOptions = { ...options };
+      newOptions.search = e.target.value;
+      return newOptions;
+    });
+    // setSubmitted(true);
```
Comment (nerfZael) [inline packages/verifier/client/web-ui/src/App.tsx:101]: what's this?

File: packages/verifier/client/web-ui/src/App.tsx [hunk 4]
```diff
@@ -60,19 +53,31 @@ function App(): JSX.Element {
...
+          console.log("2");
+          setLogs(logs);
+        })
+        .catch((error) => {
+          console.log(error);
+        });
+    };
+
+    logFetcher();
+    console.log(3);
```
Comment (nerfZael) [inline packages/verifier/client/web-ui/src/App.tsx:73]: mmm, yes, I see
:P

Comment (Niraj-Kamdar) [reply packages/verifier/client/web-ui/src/App.tsx:73]: debug leftover oops!

**Review by nerfZael — APPROVED** [review]

**Comment by nerfZael** [comment]
Nice, looks a lot cleaner!

**Pull request merged**